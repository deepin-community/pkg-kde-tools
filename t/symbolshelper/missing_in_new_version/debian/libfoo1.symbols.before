# SymbolsHelper-Confirmed: 0.1 amd64
libfoo.so.1 libfoo1 #MINVER#
 _Z3barv 0.1
 _Z3foov 0.1
