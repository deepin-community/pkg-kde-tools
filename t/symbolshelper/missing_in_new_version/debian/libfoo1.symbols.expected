# SymbolsHelper-Confirmed: 1.0 amd64
libfoo.so.1 libfoo1 #MINVER#
#MISSING: 1.0# _Z3barv 0.1
 _Z3foov 0.1
